### PR TITLE
Update our browserslist config per "Best Practices"

### PR DESCRIPTION
**Pull request by bradenmacdonald**
Our browserslist config is currently not following the [Best Practices](https://github.com/browserslist/browserslist?tab=readme-ov-file#best-practices) that the browserslist project recommends.

In particular, they say:

> Don’t remove browsers just because you don’t know them. Opera Mini has 100 million users in Africa and it is more popular in the global market than Microsoft Edge. Chinese QQ Browsers has more market share than Firefox and desktop Safari combined.

Here is the list of supported browsers (from `npm run supported`), before and after this change:

```diff
 and_chr 131
 and_ff 132
+and_qq 14.9
+and_uc 15.5
+android 131
 chrome 131
 chrome 130
+chrome 129
+chrome 128
+chrome 127
+chrome 126
+chrome 125
+chrome 124
+chrome 109
 edge 131
 edge 130
+edge 129
+edge 128
 firefox 132
 firefox 131
+firefox 130
+firefox 129
+firefox 115
 ios_saf 18.1
 ios_saf 18.0
 ios_saf 17.6-17.7
 ios_saf 17.5
 ios_saf 17.4
 ios_saf 17.3
 ios_saf 17.2
 ios_saf 17.1
 ios_saf 17.0
 ios_saf 16.6-16.7
 ios_saf 16.5
 ios_saf 16.4
 ios_saf 16.3
 ios_saf 16.2
 ios_saf 16.1
 ios_saf 16.0
+ios_saf 15.6-15.8
+kaios 3.0-3.1
+kaios 2.5
+op_mini all
+op_mob 80
+opera 114
+opera 113
+opera 112
 safari 18.1
 safari 18.0
 safari 17.6
 safari 17.5
 safari 17.4
 safari 17.3
 safari 17.2
 safari 17.1
 safari 17.0
+safari 16.6
+safari 15.6
+samsung 26
+samsung 25
```

I don't have a strong opinion on this change, but I do think it makes sense to follow the best practices if there's little or no cost to doing so.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @bradenmacdonald!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?



This repository is currently unmaintained.

To get help with finding a technical reviewer, tag the community contributions project manager for this PR in a comment and let them know that your changes are ready for review:

1. On the right-hand side of the PR, find the Contributions project, click the caret in the top right corner to expand it, and check the "Primary PM" field for the name of your PM.
2. Find their GitHub handle [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3548807177/Community+Contributions+Project+Manager#Current-OSPR-Project-Managers).


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Review by adamstankiewicz — APPROVED** [review]

**Review by brian-smith-tcril — APPROVED** [review]

**Review by arbrandes — APPROVED** [review]
👍🏼

**Comment by bradenmacdonald** [comment]
Something went wrong with the `release` CI action after I merged this, but I'm not sure what it means (404 error on GitHub API)

**Comment by brian-smith-tcril** [comment]
I retried the job and it looks like it worked this time.

**Comment by brian-smith-tcril** [comment]
Hmm, so on the re-run it didn't find any changes which is why it passed. It seems publishing to `npm` is working properly but publishing to GH releases isn't.

The latest GH release is from April 2023 https://github.com/openedx/browserslist-config/releases/tag/v1.2.0

The latest `npm` release is from half an hour ago https://www.npmjs.com/package/@edx/browserslist-config/v/1.4.0

**Comment by brian-smith-tcril** [comment]
Made an issue for the GH releases failing https://github.com/openedx/browserslist-config/issues/17

**Comment by bradenmacdonald** [comment]
Do we really use/need the GitHub release?

**Comment by brian-smith-tcril** [comment]
> Do we really use/need the GitHub release?

We don't *need* them, but the automatic changelogs in there can be nice.

**Comment by bradenmacdonald** [comment]
@brian-smith-tcril We may want to revert this. I've found that when pulling it into Paragon, it seems to undo the changes from https://github.com/openedx/paragon/pull/3284 , i.e. it adds a ton of unnecessary backwards compatibility shims back into every icon file.

**Pull request merged**
